04-reporting.Rmd

```diff
@@ -543,7 +701,7 @@ knitr::kable(print_classes, format = "html", escape = FALSE)
 <span style="font-size:50%;"><sup>\*</sup> Not fully tested, don't trust blindly!</span>
 
 
-```{r formatting-process-diagram, fig.cap = "Process diagram illustrating the use of `apa_print()` methods and `apa_table()` to format results from statistical analyses according to APA guidelines. The formatted text and tables can be reported in a manuscript.", fig.align = "center", echo = FALSE}
+```{r formatting-process-diagram, fig.cap = "Process diagram illustrating the use of `apa_print()` methods and `apa_table()` to format results from statistical analyses according to APA guidelines. The formatted text and tables can be reported in a manuscript.", fig.align = "center", echo = FALSE, eval = FALSE}
```

Comment:
Not sure, why you removed the figure, but I think it may be a useful overview for more R-savy users. Do you think it would hurt to keep it?

Reply:
added it back, again